confirm_the_ending.js

```diff
@@ -0,0 +1,14 @@
+function confirmEnding(str, target) {
+  
+ var first = str.length, last = target.length;
+  
+ if(str.substring(first - last, first) === target){
```

Comment:
mozesz zrobic po prostu:
```javascript
return (str.substring(first - last, first) === target);
```

dodatkowo - przyjmujemy, ze str to string. jezeli przypisujesz do niego wartosc typu boolean to robisz mocne zaklamanie na typach i wprowadzasz niepewnosc do kodu. postaraj sie tego unikac.